lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -174,7 +178,23 @@ def get(self):
                     templates[path] = template
 
         try:
+            oidc_secret = self.server_config.get("authentication", "secret")
+        except NoOptionError:
+            oidc_secret = ""
+        try:
+            client = self.server_config.get("authentication", "client")
+            realm = self.server_config.get("authentication", "realm")
+            issuer = (
+                self.server_config.get("authentication", "server_url")
+            ) + f"/{realm}"
+            auth_object = {
+                "client": client,
+                "realm": realm,
+                "issuer": issuer,
+                "secret": oidc_secret,
+            }
             endpoints = {
+                "authentication": auth_object,
```

Comment:
Consider in-lining the subobject:
```
            endpoints = {
                "authentication": {
                    "client": client,
                    "realm": realm,
                    "issuer": issuer,
                    "secret": oidc_secret,
                },
                ...
```

Ditto in `lib/pbench/test/unit/server/test_endpoint_configure.py`, too.